dub.sdl

```diff
@@ -0,0 +1,14 @@
+name "har"
+description "A human-readable archive format"
+authors "Jonathan Marler"
+copyright "Copyright © 2018, Jonathan Marler"
+license "public domain"
+targetType "executable"
+dependency "har:library" version="*"
+sourceFiles "harmain.d"
+
+subPackage {
```

Comment:
* how about adding subPackage for the other things (examples, test)
* using a `source` subdirectory is common practice eg:
`source/archive/har.d`

Reply:
done